ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -261,6 +269,15 @@ extension MSFListView {
                 }
             })
             .buttonStyle(ListCellButtonStyle(tokens: tokens, layoutType: layoutType))
+            if let children = cell.children, isExpanded == true {
+                ForEach(children, id: \.self) { child in
```

Comment:
I noticed these dividers for the sub items and I think this is probably not the way it should look.
Can you check with Victor on that and hide them while you don't have clear design guidelines? (creating a GitHub issue if the final look won't be merged as part of this PR as well)

![Screen Shot 2021-01-14 at 6 43 53 PM](https://user-images.githubusercontent.com/68076145/104674458-8ddcaf00-5698-11eb-9f39-f16558b186b9.png)

Reply:
Lack of modifiers + Apple fixing UITableView accessor -- created an iOS 14 check and changed to a ScrollView, otherwise uses a List with the UITableView modifier.